cogs/christmas_2021.py

```diff
@@ -69,8 +69,11 @@ async def christmas(self, ctx):
         """View Christmas event information."""
 
         member = await self.bot.mongo.fetch_member_info(ctx.author)
+        
+        def christmas_2021_ec(self):
+            return random.choice([0x9ECFFC, 0xDE2E43, 0x79B15A])
 
-        embed = self.bot.Embed(color=0xE67D23)
+        embed = self.bot.Embed(color=self.christmas_2021_ec())
```

Comment:
Is the function *really* neccesary here? Imo there's nothing wrong with simply passing the random.choice directly into color

Reply:
we did try it but for some reason it doesn't randomize everytime the command is used, but instead everytime the cog is loaded? i dont know if i did anything wrong during the testing tho

Reply:
i tried it again and it works, weird. thanks!

Reply:
Sorry for the inconvenience, it worked when we tested it out again.